packages/frameworks/src/frameworks.ts

```diff
@@ -1960,6 +1960,49 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'FastHTML (Experimental)',
+    slug: 'fasthtml',
+    demo: 'https://fasthtml-template.vercel.app',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/fasthtml.svg',
```

Comment:
I can not find a square logo, but we should get one. Alternatively, we can place the generic Python logo here for now if we must use a square logo.